cmd/neofs-cli/modules/tree/add.go

```diff
@@ -0,0 +1,119 @@
+package tree
+
+import (
+	"crypto/sha256"
+	"encoding/hex"
+	"errors"
+	"fmt"
+	"strings"
+
+	"github.com/nspcc-dev/neofs-node/cmd/neofs-cli/internal/common"
+	"github.com/nspcc-dev/neofs-node/cmd/neofs-cli/internal/commonflags"
+	"github.com/nspcc-dev/neofs-node/cmd/neofs-cli/internal/key"
+	"github.com/nspcc-dev/neofs-node/pkg/services/tree"
+	cid "github.com/nspcc-dev/neofs-sdk-go/container/id"
+	"github.com/spf13/cobra"
+)
+
+var addCmd = &cobra.Command{
+	Use:   "add",
+	Short: "Add a node to the tree service",
+	Run:   add,
+	PersistentPreRun: func(cmd *cobra.Command, _ []string) {
+		commonflags.Bind(cmd)
+	},
+}
+
+func initAddCmd() {
+	commonflags.Init(addCmd)
+	initCTID(addCmd)
+
+	ff := addCmd.Flags()
+	ff.StringSlice(metaFlagKey, nil, "Meta pairs in the form of Key1=[0x]Value1,Key2=[0x]Value2")
+	ff.Uint64(pidFlagKey, 0, "Parent node ID")
+
+	_ = cobra.MarkFlagRequired(ff, commonflags.RPC)
+}
+
+func add(cmd *cobra.Command, _ []string) {
+	pk := key.GetOrGenerate(cmd)
+
+	var cnr cid.ID
+	err := cnr.DecodeString(cmd.Flag(cidFlagKey).Value.String())
+	common.ExitOnErr(cmd, "decode container ID string: %w", err)
+
+	tid := cmd.Flag(tidFlagKey).Value.String()
+	if tid == "" {
+		common.ExitOnErr(cmd, "", errors.New("empty tree ID"))
+	}
```

Comment:
Can we just make the flag required? I believe tree service supports empty names now.

Reply:
> I believe tree service supports empty names now

ok, but what for? dropped that limitation